File: Example/Shared/Settings/UIFont+THRFonts.m
//
//  UIFont+THRFonts.m
//  Example
//
//  Created by Brooma Service on 20.12.2019.
//  Copyright © 2019 Threads. All rights reserved.
//

#import "UIFont+THRFonts.h"

@implementation UIFont (THRFonts)

+ (UIFont *)latoRegularOfSize:(CGFloat)fontSize {
    return [UIFont fontWithName:@"Lato-Regular" size:fontSize];
}

+ (UIFont *)latoSemiboldOfSize:(CGFloat)fontSize {
    return [UIFont fontWithName:@"Lato-Semibold" size:fontSize];
}

+ (UIFont *)latoLightOfSize:(CGFloat)fontSize {
    return [UIFont fontWithName:@"Lato-Light" size:fontSize];
}

+ (UIFont *)latoMediumOfSize:(CGFloat)fontSize {
    return [UIFont fontWithName:@"Lato-Medium" size:fontSize];
}

@end

File: Example/ExampleSwift/Integration/Attributes/UIFont+THRFonts.swift
//
//  UIFont+THRFonts.swift
//  ExampleSwift
//
//  Created by Brooma Service on 23.04.2020.
//  Copyright © 2020 Threads. All rights reserved.
//

import Foundation
import UIKit

extension UIFont {
    
    public class func latoRegular(ofSize: CGFloat) -> UIFont {
        return UIFont.init(name: "Lato-Regular", size: ofSize) ?? UIFont.systemFont(ofSize: ofSize)
    }

    public class func latoSemibold(ofSize: CGFloat) -> UIFont {
        return UIFont.init(name: "Lato-Semibold", size: ofSize) ?? UIFont.systemFont(ofSize: ofSize)
    }

    public class func latoLight(ofSize: CGFloat) -> UIFont {
        return UIFont.init(name: "Lato-Light", size: ofSize) ?? UIFont.systemFont(ofSize: ofSize)
    }

    public class func latoMedium(ofSize: CGFloat) -> UIFont {
        return UIFont.init(name: "Lato-Medium", size: ofSize) ?? UIFont.systemFont(ofSize: ofSize)
    }

}
